prebuild/macOS/binding.gyp

```diff
@@ -34,6 +34,8 @@
         '<!@(pkg-config pangocairo --libs)',
         '<!@(pkg-config freetype2 --libs)',
         '<!@(pkg-config librsvg-2.0 --libs)',
+        '<!@(pkg-config libjpeg --libs)',
+        '-L/opt/homebrew/lib',
```

Comment:
Running `brew --prefix` or `echo $HOMEBREW_PREFIX` will generate `/opt/homebrew` on macOS ARM (Apple Silicon) CPUs but will generate `/usr/local` on Intel Macs.  https://docs.brew.sh/Installation

We should support both paths.